bundle/edu.gemini.p1monitor/src/main/scala/edu/gemini/p1monitor/P1MonitorMailer.scala

```diff
@@ -134,7 +134,7 @@ class P1MonitorMailer(cfg: P1MonitorConfig) {
 
   private def getInstrumentsString(prop: Proposal): String = prop.observations.map {
       obs => obs.blueprint match {
-        case Some(bp: GeminiBlueprintBase) => s"${bp.instrument.site.name} (${bp.instrument.id})"
```

Comment:
I'm guessing the instrument site is used elsewhere?  Is it not a worry that the two could be inconsistent?  If we need to do this for now to avoid making waves, I understand but we should probably at least add an issue to evaluate this because it seems like we're not modeling the situation correctly.  Perhaps for visitor instruments the site needs to be supplied when an instance is created or something.